shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -240,13 +244,96 @@ bool FlutterWindowsEngine::Run() {
   return Run("");
 }
 
+std::pair<std::string, int> GetReleaseVersionAndBuildNumber() {
```

Comment:
I think a more common pattern for a C/c++ api, would be to pass in a struct that you fill, and then have it return an error value if needed?

struct Results {
  std::string release_version;
  int build_number;
}

int GetReleaseVersionAndBuildNumber(Results& results);

Results results;
int error = GetReleaseVersionAndBuildNumber(results);
if (error != 0) {
  // do the error thing
}

I suspect you'll find it's a bit too cute to put the release version and error in the same storage.

Reply:
The int was the build number, but I've created a struct to encapsulate the release version + build number combo